src/main/java/domain/GamePlayer.java

```diff
@@ -0,0 +1,82 @@
+package domain;
+
+import io.Message;
+import io.Printer;
+import io.Receiver;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class GamePlayer {
+    private static final int WINNER_CONDITION = 4;
+
+    private final Printer printer;
+    private final Receiver receiver;
+    private final Generator generator;
+
+    public GamePlayer() {
+        this.printer = new Printer();
+        this.receiver = new Receiver();
+        this.generator = new Generator();
+    }
+
+    void moveAfterJudgement(Car car, int randomNumber) {
+        if (judgeToMove(randomNumber)) {
+            car.moveForward();
+        }
+    }
+
+    private boolean judgeToMove(int randomNumber) {
+        return randomNumber >= WINNER_CONDITION;
+    }
+
+    List<Car> inputNames() {
+        String messageCode = Message.GeneralMessages.INPUT_NAMEOFCAR.getMessage();
+        printer.printMessages(messageCode);
+        List<String> names = new ArrayList<>(Arrays.asList(receiver.receiveName()));
```

Comment:
```suggestion
        List<String> names = Arrays.asList(receiver.receiveName());
```
`new ArrayList<>()`를 안해줘도 괜찮습니다.